deploy/helm/values/uat.yaml

```diff
@@ -29,11 +29,14 @@ postgresql:
     requests:
       cpu: 250m
       memory: 256Mi
+  auth:
+    enablePostgresUser: true
+    postgresPassword: postgres
+    database: cfe_civil
   postgresqlUsername: postgres
```

Comment:
we are treating postgresqlUsername as a hangover - it's always 'postgres'. Now I understand how it works, we could probably change to using a user/password combo rather than the postgres one